src/components/alert/alert.stories.spec.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Alert from './Alert';
+
+export default {
+  component: Alert,
+  title: 'Alert',
+};
+
+export const Main = () => (
+  <Alert />
```

Comment:
This could use some more love
Since you're not passing `active` and `children` nothing is visible [(result here)](https://www.chromatic.com/pullrequest?activeElementId=comparison-test-key-6241eaa504f8fff940c46360-1200&appId=611620e7b1a00e003cde6c49&number=2057&view=changes)

Reply:
Seems to be forgotten indeed. I will give it the love it deserves!